src/main/java/de/tum/in/www1/artemis/web/rest/ModelingSubmissionResource.java

```diff
@@ -329,18 +329,16 @@ else if (!authCheckService.isAtLeastInstructorForExercise(exercise)) {
     @PreAuthorize("hasAnyRole('USER', 'TA', 'INSTRUCTOR', 'ADMIN')")
     public ResponseEntity<ModelingSubmission> getSubmissionForModelingEditor(@PathVariable Long participationId) {
         StudentParticipation participation = participationService.findOneWithEagerSubmissionsAndResults(participationId);
-        if (participation == null) {
```

Comment:
This code is not necessary because `findOneWithEagerSubmissionsAndResults` throws an exception if there is no entity found.